config/e2e/config.yaml

```diff
@@ -7,6 +7,6 @@ data:
     kuberay:
       rayDashboardOAuthEnabled: false
       ingressDomain: "kind"
-      certGeneratorImage: quay.io/project-codeflare/ray:latest-py39-cu118
+      certGeneratorImage: quay.io/project-codeflare/ray:2.20.0-py39-cu118
```

Comment:
Can this be UBI9 image. A Ray image is not needed to generate the TLS certs.

Reply:
I think the default has been set to `registry.redhat.io` for downstream but it requires authentication. The default, at least for the e2e, should be the RH access registry.

Reply:
Testing this approach on https://github.com/project-codeflare/codeflare-operator/pull/568
Required upgrade of Ray image in codeflare-common repo